src/inventory/bmc.go

```diff
@@ -18,7 +21,10 @@ func newBMC(dependencies IDependencies) *bmc {
 }
 
 func (b *bmc) modprobe (module string) {
```

Comment:
What about returning error from the function?

I understand that right now we ignore the error, but its best for the user of the function to decide that and not the function itself.

Reply:
OK, but I think that the flow of getting the bmcAddress should not be affected if loading modules returns an error.  It should continue in the regular flow.

Reply:
That is fine, but it is up to the caller to decide